packages/node_modules/@ciscospark/widget-message/src/container.js

```diff
@@ -736,6 +742,7 @@ export default connect(
     setTyping,
     showScrollToBottomButton,
     updateHasNewMessage,
-    updateWidgetState
+    updateWidgetState,
+    getFeature
```

Comment:
Can you add it to the injected prop types as well?

Reply:
Yup. Done